### ISSUE TITLE
Batch effect correction in PHATE

### ISSUE THREAD

**Issue by pmb59** [KrishnaswamyLab/PHATE#76]
Hello

What is the recommended (if any) method for batch correction before running the function `phate`?

thanks
Pedro

**Comment by scottgigante** [comment]
Hi Pedro,

You are free to use whatever batch correction you like -- PHATE doesn't depend on any particular kind of input data. However, we do have an experimental / undocumented batch correction built into PHATE which you can run by using the keyword arguments `sample_idx=[...]` (a list of batch labels, one for each row in `data`) and `kernel_symm='mnn'`. For example:

```
data_batch1 = ...
data_batch2 = ...
data_combined, sample_labels = scprep.utils.combine_batches([data_batch1, data_batch2], ['batch1', 'batch2'])
data_phate = phate.PHATE(sample_idx=sample_labels, kernel_symm='mnn').fit_transform(data)
```

**Comment by pmb59** [comment]
Hi Scott

thanks. I am running the R version of PHATE. Is there any implementation there?

best
Pedro

**Comment by scottgigante** [comment]
I think if you passed the same inputs (`sample_idx=list(...), kernel_symm='mnn'`) to the R implementation they should go through to the Python backend. But if other implementations of MNN work for you, then great! (ours is a slightly different algorithm but the underlying idea is the same.)
